src/apps/vpn/ui/screens/home/ViewServers.qml

```diff
@@ -16,7 +16,7 @@ import compat 0.1
 Item {
     id: root
     objectName: "viewServers"
-
+    Accessible.name: qsTrId("vpn.servers.selectLocation")
```

Comment:
When does this get read?

Reply:
Read when the screen reader shortcuts are used to move to the previous or next item, and that navigation reaches the 'Select Location' text.